mappings/net/minecraft/level/dimension/SkyDimension.mapping

```diff
@@ -0,0 +1 @@
+CLASS ay net/minecraft/level/dimension/SkyDimension
```

Comment:
hmmm... maybe `TheSky`

Reply:
or `Skylands`

Reply:
Is just `Sky` okay?

Reply:
`Sky` could be confused with the actual rendered sky